--- mkconfig.orig	2008-11-19 15:34:42.000000000 -0200
+++ mkconfig.rb	2008-06-06 07:39:57.000000000 -0300
@@ -75,7 +75,7 @@
     next if /^\$ac_\w+$/ =~ val
     next if $install_name and /^RUBY_INSTALL_NAME$/ =~ name
     next if $so_name and /^RUBY_SO_NAME$/ =~  name
-    next if /^(?:X|(?:MINI|RUN)RUBY$)/ =~ name
+    #next if /^(?:X|(?:MINI|RUN)RUBY$)/ =~ name
     if /^program_transform_name$/ =~ name and /^s(\\?.)(.*)\1$/ =~ val
       next if $install_name
       sep = %r"#{Regexp.quote($1)}"
